Update README.md (#62)
web-platform-tests/wpt-metadata: README.md

  - e.g. `chrome`, `safari-12`, or `firefox-63.0-linux`
- If omitted, the link will apply to all products in the directory.
- Test path is relative to the current directory, so will typically be just a filename.
- Test result status is a status as defined in the [wpt.fyi codebase](https://github.com/web-platform-tests/wpt.fyi/blob/master/shared/statuses.go).

## How to contribute to wpt-metadata repository
You can contribute to the wpt-metadata repo by sending out a PR directly or through the Triage Metadata API available for trusted third parties.

### Triage Metadata API for trusted third parties
To use the Triage Metadata API, you first need to sign in to [wpt.fyi](https://wpt.fyi/) (top-right corner; 'Sign in with GitHub'). For more information on wpt.fyi login, see [here](https://docs.google.com/document/d/1iRkaK6cGgXp3DKbNbPMVsYGMaOHO-5CfqEuLPUR_2HM).

> The logged-in user also needs to belong to the ['web-platform-tests' GitHub organization](https://github.com/orgs/web-platform-tests/people). To join, please [file an issue](https://github.com/web-platform-tests/wpt/issues/new?).

Reviewer: ```suggestion
The logged-in user also needs to belong to the ['web-platform-tests' GitHub organization](https://github.com/orgs/web-platform-tests/people). To join, please [file an issue](https://github.com/web-platform-tests/wpt/issues/new?), including the reason you need access to the Triage Metadata API.
```

Once logged in, you can send a request to https://wpt.fyi/api/metadata/triage to triage metadata. This endpoint only accepts PATCH requests, with a body that contains a valid triage JSON object. The JSON object is a flattened YAML `Links` structure that is keyed by test name [Test path](https://docs.google.com/document/d/1oWYVkc2ztANCGUxwNVTQHlWV32zq6Ifq9jkkbYNbSAg/edit#heading=h.t7ysbpr8er1y); see below for an example.

> /api/metadata/triage returns the url of a PR that is created in the wpt-metadata repo.

Reviewer: ```suggestion
`/api/metadata/triage` returns the url of a PR that is created in the wpt-metadata repo.
```

- If omitted, the link will apply to all products in the directory.
- Test path is relative to the current directory, so will typically be just a filename.
- Test result status is a status as defined in the wpt.fyi codebase.

## How to contribute to wpt-metadata repository

> If you are a memeber of the web-platform-tests org, you can contribute to the wpt-metadata repo by sending out a PR directly or through a UI-based approach on wpt.fyi. We also have a Triage Metadata API available for trusted third parties

Reviewer: I don't think this is correct:

- Can't anyone send a PR to this repository? Don't need to be a member of wpt org
- We don't have a UI yet, let's not mention it

### Triage through wpt.fyi UI
For information about how to use the triage service on wpt.fyi, see here - UPCOMING

### Triage Metadata API for trusted third parties

> In order to use the Triage Metadata API, first you need to log in on [wpt.fyi](https://wpt.fyi/) through the GitHub OAuth integration. For more information on wpt.fyi login, see [here](https://docs.google.com/document/d/1iRkaK6cGgXp3DKbNbPMVsYGMaOHO-5CfqEuLPUR_2HM)

Reviewer: ```suggestion
To use the Triage Metadata API, you first need to sign in to [wpt.fyi](https://wpt.fyi/) (top-right corner; 'Sign in with GitHub'). For more information on wpt.fyi login, see [here](https://docs.google.com/document/d/1iRkaK6cGgXp3DKbNbPMVsYGMaOHO-5CfqEuLPUR_2HM).
```

(Note also that these instructions will be wrong until prod ships sign-in. When is that happening?)

Author: The sign-in has shipped in the last release, although the next release is needed for api/metadata/triage to work end-to-end

Reviewer: I do not see a sign-in button on https://wpt.fyi, only https://staging.wpt.fyi.

Looking at https://wpt.fyi/flags, it looks like the feature may be behind a flag. Until it is available for everyone by default, it has not shipped.

Author: Technically it is available by default as long as a user enables that flag themselves? since https://wpt.fyi/flags is user-side flags?

Reviewer: Right, but how does a user know to go find that (hidden) page and enable that flag?

Author: I see. I won't submit this document until login is fully working in prod, which is most likely the next release

> The logged-in user also needs to be a part of the web-platform-tests org. To join, please file an issue or contact us directly.

Reviewer: ```suggestion
The logged-in user also needs to belong to the ['web-platform-tests' GitHub organization](https://github.com/orgs/web-platform-tests/people). To join, please file an issue or contact us directly.
```
Where should they file an issue? (I suggest linking to the correct create URL, e.g. is it https://github.com/web-platform-tests/wpt/issues/new?)

How should they contact us directly? Is there an email address we should link them.

Author: Linked. I checked wpt.fyi and the wpt repository and looks like we don't give out email contacts. I will do the same here

> Upon a login, you can send a request to /api/metadata/triage endpoint to triage metadata. This endpoint only accepts PATCH request and JSON object body. The JSON object is a flattened YAML `Links` structure which is keyed by test name [Test path].

Reviewer: ```suggestion
Once logged in, you can send a request to https://wpt.fyi/api/metadata/triage to triage metadata. This endpoint only accepts PATCH requests, with a body that contains a valid triage JSON object. The JSON object is a flattened YAML `Links` structure that is keyed by test name [Test path]; see below for an example.
```

I don't understand what 'test name [Test path]' means reading this.

Author: [Test path] is from the line above, https://github.com/web-platform-tests/wpt-metadata/pull/62/files#diff-04c6e90faac2675aa89e2176d2eec7d8R13. I thought it would make it more clear.
